static/public/index.html

```diff
@@ -4443,6 +4448,3396 @@ <h3 class="text-subtitle font-bold text-dark">Pickly</h3>
+                    />
+                  </svg>
+                  <div class="relative w-full mt-xxs">
+                    <input
+                      class="border border-accent bg-accent-shd7 bg-opacity-10 py-xsvv px-m text-sm font-normal text-accent w-full rounded-md focus:outline-none"
+                      type="text"
+                      placeholder="Add option"
+                      class=""
+                    />
+                  </div>
+                </div>
+              </div>
+              <div class="h-31xl md:hidden w-full bg-transparent"></div>
+            </div>
+            <div
+              class="misc-box md:bg-white md:shadow-soft md:rounded-b-md border-b border-grey-shd6 md:border-none flex p-m md:mb-10"
+            >
+              <div class="flex justify-between w-21xl md:w-37xl">
+                <div class="flex items-center justify-between w-full md:w-auto">
+                  <label class="toggle-box">
+                    <input type="checkbox" class="checkbox" />
+                    <div class="toggler toggler-small toggle-org">
+                      <div class="circle circle-small"></div>
+                    </div>
+                  </label>
+                  <p class="ml-xs text-xs text-dark-grey font-normal">
+                    Hide my identity <span class="text-grey-shd6 ml-m">|</span>
+                    <span class="ml-m">Privacy:</span>
+                    <span class="ml-xs text-sm font-medium">Public</span>
+                  </p>
+                  <svg
+                    class="ml-xxs"
+                    width="16"
+                    height="16"
+                    viewBox="0 0 16 16"
+                    fill="none"
+                    xmlns="http://www.w3.org/2000/svg"
+                  >
+                    <path
+                      d="M10.862 6.19534L7.99998 9.05734L5.13798 6.19534L4.19531 7.13801L7.99998 10.9427L11.8046 7.13801L10.862 6.19534Z"
+                      fill="#5B6366"
+                    />
+                  </svg>
+                </div>
+                <button
+                  class="text-sm hidden md:inline-block font-medium text-white bg-primary opacity-24 pointer-events-none py-xxsv px-m rounded-full"
+                >
+                  Post
+                </button>
+              </div>
+            </div>
+            <div class="flex md:hidden p-m mb-10 w-full justify-between">
+              <button
+                class="bg-white text-error text-sm font-medium py-xxsv px-xl rounded-full"
+              >
+                Cancel
+              </button>
+              <button
+                class="bg-primary opacity-24 text-white font-medium text-sm py-ccsv px-24 rounded-full"
+              >
+                Post
+              </button>
+            </div>
+          </div>
+        </div>
+
+        <div class="parent">
+          <div class="title-header">
+            <h3 class="title text-lg">Mini Survey - Default</h3>
```

Comment:
1. Add option group is not identical as design 
2. spacing between option is not like design